clang/lib/Driver/ToolChains/PS4CPU.cpp

```diff
@@ -237,7 +237,10 @@ void tools::PS5cpu::Linker::ConstructJob(Compilation &C, const JobAction &JA,
   if (!D.SysRoot.empty())
     CmdArgs.push_back(Args.MakeArgString("--sysroot=" + D.SysRoot));
 
-  if (Args.hasArg(options::OPT_pie))
+  // Default to PIE for non-static main components.
```

Comment:
While the generic ABI refers to executable and shared object files as "components", "main component" isn't used.

main executable or just executable

Reply:
Done!